packages/table/src/components/toolbar/ExportToExcelBtn.tsx

```diff
@@ -0,0 +1,18 @@
+import Button from '@equinor/fusion-react-button';
+import { useTableContext } from '../../TableProvider';
+import { useInstance } from 'table/src/plugins/excel/useInstance';
+/**
+ * Component for the `Toolbar` component.
+ * Only shown if the `hideExportBtn` prop in `Toolbar` is set to `false`.
+ * Uses {@link useInstance the excel plugin} to handle all exporting logic.
```

Comment:
Do we want this type of comment where we use `@link`? Have to import the thing I want to link to.